openslides/core/models.py

```diff
@@ -152,6 +163,25 @@ def need_full_update_for(self, collection_element):
         return True
 
 
+class ProjectionDefault(RESTModelMixin, models.Model):
```

Comment:
Where do you need the default projector id? On server side, you can still get it as you wrote if we use two models. On client side you can add a short projector method to do the thing easily.